socialwarehouse/geo/migrations/0003_address_charfield_null_to_blank_default.py

```
@@ -0,0 +1,198 @@
+    "cd_geoid",
+    "sldl_geoid",
+    "sldu_geoid",
+]
+
+
+# Pre-AlterField data backfill. Single SQL statement with all 29 UPDATEs.
+# COALESCE is not used because we want explicit NULL→'' semantics, not
+# coalesce-on-read. Each UPDATE is a no-op for rows that already have
+# non-NULL values.
+_BACKFILL_SQL = "\n".join(
+    f"UPDATE sw_geo_address SET {field} = '' WHERE {field} IS NULL;"
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)